.github/workflows/update-helm-docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+# Check if helm-docs is installed
+if ! command -v helm-docs &>/dev/null; then
+    echo "helm-docs could not be found. Installing it..."
+    go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
+fi
+
+# Navigate to the charts directory
+cd ../../helm
+
+helm-docs --chart-search-root="." --template-files=VALUES.md.gotmpl --output-file=../README.md
+
+echo "Helm documentation updated successfully!"
```

Comment:
Add new line and change file location outside the "workflows" directory

Reply:
the update-helm-docs.sh or the readme?